_posts/<spaceranger-23>/2020-09-04-first-reflection-<spaceranger-23>.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+Author: Spaceranger-23
```

Comment:
the `author:` key and your github username should be lower case here. Our program isn't smart enough to know what you mean otherwise.

Reply:
username fix has been resolved!